packages/plugins/other/jsdoc/src/index.ts

```diff
@@ -181,6 +187,12 @@ export const plugin: PluginFunction<RawDocumentsConfig> = (schema, documents) =>
         return createDocBlock([createDescriptionBlock(node), `@typedef {*} ${node.name}`]);
       },
     },
+    SchemaDefinition: {
```

Comment:
I think this is extra

Reply:
Ah you're right! I updated the existing definition to return `null` instead; I found that returning an empty string causes a new line to be generated unnecessarily.

Also: not sure if I created a changeset correctly as I've never used it before. Let me know if I need to change anything!